client/src/cmdscript.c

```diff
@@ -403,18 +403,23 @@ static int CmdScriptRun(const char *Cmd) {
         // hook Proxmark3 API
         PyImport_AppendInittab("_pm3", PyInit__pm3);
 #endif
+#if PY_MINOR_VERSION >= 10
```

Comment:
Filtering only with the minor version might not be a good idea.

Reply:
You have right! Added MAJOR too, thanks!